### DISCUSSION TITLE
How to show buffer(tabs) even when only 1 file is open?

### DISCUSSION THREAD

**Discussion by RHOOPH** [LazyVim/LazyVim#1330] [Q&A] [answered]
This is how buffers(tabs) look when more than 1 file is open
![image](https://github.com/LazyVim/LazyVim/assets/16752005/7614e037-118f-4e93-80cf-d5556006bc0b)

But when only 1 file is open, it looks like this
![image](https://github.com/LazyVim/LazyVim/assets/16752005/d6e3c24f-bb28-4b3a-ab6b-eac03447a6ee)

This causes problem as i can't really tell if a file is modified or not when only 1 buffer is open. How can always show the buffer(tabs) ?

**Comment by dpetka2001** [comment, marked as answer]
```lua
  {
    "akinsho/bufferline.nvim",
    opts = {
      options = {
        always_show_bufferline = true,
      },
    },
  },
```
in a file inside `~/.config/nvim/lua/plugins` directory. Make sure you also add a `return` statement in that file. I'm not including it here, since you could just put the above snippet in an existing file.

**Reply by bitzcoder** [reply]
But now the [no name] bufferline shows up even on the dashboard. How do I hide it? I have tried with the autocmd below, but it doesn't seem to work:
```lua
vim.api.nvim_create_autocmd("User", {
  pattern = "SnacksDashboardOpened",
  callback = function()
    vim.opt.showtabline = 0
    vim.api.nvim_create_autocmd("BufUnload", {
      buffer = vim.api.nvim_get_current_buf(),
      callback = function()
        vim.opt.showtabline = 2
      end,
    })
  end,
})

```
